src/containers/AccountModal/AccountModal.tsx

```diff
@@ -60,7 +58,11 @@ const AccountModal = () => {
     if (!!viewParam && !loading && !user && !isPublicView) {
       toLogin();
     }
-  }, [viewParam, loading, user, isPublicView, toLogin]);
+  }, [viewParam, navigate, location, loading, isPublicView, user, toLogin]);
```

Comment:
I think these deps are incorrect after fixing the merge conflicts. The `navigate` and `location` deps aren't needed anymore